go-controller/pkg/config/config.go

```diff
@@ -95,6 +95,7 @@ var (
 	// HybridOverlay holds hybrid overlay feature config options.
 	HybridOverlay = HybridOverlayConfig{
 		RawClusterSubnets: "10.132.0.0/14/23",
+		VXLANPort: 4789, // 4789 is the default IANA-assigned VXLAN UDP port number
```

Comment:
Do we need to hardcode this twice? Could node.go not pick that up from config.go?

Reply:
Generally speaking, yes, I prefer have one hard-coded location for these constants. The pattern used in config.go seem to indicate the constants were coded all over, so I tried to follow the existing pattern. That said, I found a precedence in using values from config.go from other places, so your suggestion makes sense. I will fix.